dashboard/app/views/levels/_level_group.haml

```diff
@@ -36,14 +36,7 @@
           %br/
 
         - page.levels.each_with_index do |level, index|
-          -# Fall back to retrieving the last_attempt from the parsed levelgroup last_attempt
-          - sublevel_last_attempt = parsed_last_attempt[level.id.to_s].try(:[], 'result')
```

Comment:
I ended up removing this line, as I'm fairly sure it's never used. The only time we'll use this is when current_user is nil. If that is the case, @last_attempt should always be nil.

Comments here seem to imply that it was never really sure the code was being used https://github.com/code-dot-org/code-dot-org/pull/8826/files